src/main/java/org/testeditor/fixture/commons/text/generate/RandomNameGenerator.java

```diff
@@ -0,0 +1,116 @@
+/*******************************************************************************
+ * Copyright (c) 2012 - 2018 Signal Iduna Corporation and others.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * which accompanies this distribution, and is available at
+ * http://www.eclipse.org/legal/epl-v10.html
+ *
+ * Contributors:
+ * Signal Iduna Corporation - initial API and implementation
+ * akquinet AG
+ * itemis AG
+ *******************************************************************************/
+
+package org.testeditor.fixture.commons.text.generate;
+
+import java.util.Random;
+
+public class RandomNameGenerator {
+    
+    private static final Random randomNumber = new Random(System.currentTimeMillis());
```

Comment:
ln general, randomness might be problematic for test reproducibility. One way to work around this would be to add the possibility for users to initialize the randomizer with an arbitrary, but fixed *seed* value, instead of using a quasi-random value like the current time.